### ISSUE TITLE
Editor options

### ISSUE THREAD

**Issue by akagetsu** [thebaer/MMRA#34]
Is it possible to have a sort of `.editorconfig` file to set spacing and tabbing and all that to match what you're currently using?
For example I have set my spacing for JS to 2 spaces and it formats EVERYTHING if I try to make a change..

**Comment by thebaer** [comment]
Yep, if you're familiar with the best way to set that up, please feel free to do that and reformat everything along with the config change -- we'll stick with that going forward.

**Comment by akagetsu** [comment]
Alright. I found a tool that generates an .editorconfig based on the files in the project. Seems very basic but it'll do. I'll probably add a vscode workspace file which has a lot more options than the .editorconfig

Do you have a preference on this or should I just keep it like you've started it?

**Comment by akagetsu** [comment]
maybe we could adhere to the [classic] (https://crockford.com/javascript/code.html) style-guide?
